weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -22,6 +22,7 @@
 import subprocess
 import tempfile
 import typing as t
+import metview as mv
```

Comment:
Should we do something like this here ? https://github.com/google/weather-tools/blob/main/weather_mv/loader_pipeline/regrid.py#L37.

@Piyush-Ingale ?